TrackletAlgorithm/MemoryTemplateTPROJ.h

```diff
@@ -209,7 +210,7 @@ class MemoryTemplateTPROJ
 
   void print_entry(BunchXingT bx, NEntryT index, unsigned int page = 0) const
   {
-	print_data(dataarray_[bx][(1<<NBIT_ADDR)*page+index]);
+  	print_data(dataarray_[bx][DEPTH_ADDR*page+index]);
```

Comment:
@tomalin Same comment regarding indentation.

```suggestion
    print_data(dataarray_[bx][DEPTH_ADDR*page+index]);
```

Reply:
I've replaced all tabs by spaces in MemoryTemplate.h MemoryTemplateBinnedCM.h & MemoryTemplateTPROJ.h , and then adjusted the number of spaces, such that it increases by 2, whenever, there is an additional pair of curly brackets.